books.rb

```diff
@@ -0,0 +1,16 @@
+class Books
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Please add Getters and Setters (attr_accessor) for `@rentals`, not Getters only (attr_reader) as you did.